tests/test_group.py

```diff
@@ -93,3 +102,12 @@ def test_group_invitations(self, base_url, selenium, vouched_user):
         invitations = group.invitations.invitations_list
         random_profile = randrange(len(invitations.search_invitation_list))
         assert new_member in invitations.search_invitation_list[random_profile].name
+
+    @pytest.mark.credentials
+    def test_github_non_nda_user_cannot_create_access_group(self, base_url, selenium, github_non_nda_user):
+        home_page = Home(selenium, base_url).open()
+        passcode = conftest.passcode(github_non_nda_user['secret'])
```

Comment:
You have this as a fixture, so you shouldn't need to import it from `conftest`.

Reply:
updated